index.html

```diff
@@ -123,14 +123,17 @@ <h3>
           &lt;button role="button"&gt;...&lt;/button&gt;
         </pre>
         <p>
-          Similarly, the following uses a `role=group` on a [^fieldset^] element.
+          Similarly, the following uses a `role=group` on a [^fieldset^] element, and a `role=Main` on a [^main^] element.
           This is unnecessary, because the `fieldset` element is implicitly
-          exposed as a `role=group`. Again, in practice this will likely not
-          have any unforeseen side effects.
+          exposed as a `role=group`, as is the `main` element implicity exposed as a `role=main`. Again, in practice this will likely not
+          have any unforeseen side effects to users of assistive technology, so long as the declaration of the `role` value uses <a data-cite=
+        "html/infrastructure.html#lowercase-ascii-letters">lowercase ASCII letters</a>. Please see <a href="#case-sensitivity">Case requirements for ARIA role, state and property attributes</a> for more information.
```

Comment:
```suggestion
          have any unforeseen side effects to users of assistive technology, as long as the declaration of the `role` value uses [=ascii lowercase=]. Please see <a href="#case-sensitivity">Case requirements for ARIA role, state and property attributes</a> for more information.
```

Reply:
Note: [=ascii lowercase=] is from Infa... so you need to add "infra" to the data-cite= of the body element.